File: Manuel/개인 활동/님 게임 시리즈/님 게임 홀짝/11871.cpp
#include <iostream>
#include <vector>
#define endl "\n"

using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL); cout.tie(NULL);

    int n;
    cin >> n;
    long long nim_sum = 0;

    for(int i = 0; i < n; i++)
    {
        long long p;
        cin >> p;

        if(p % 2 == 0)
        {
            nim_sum ^= p / 2 - 1;
        }
        else
        {
            nim_sum ^= p / 2 + 1;
        }
    }

    if(nim_sum != 0)
    {
        cout << "koosaga" << endl;
    }
    else
    {
        cout << "cubelover" << endl;
    }
    return 0;
}

File: Manuel/개인 활동/님 게임 시리즈/님 게임 홀짝/11871.py
n = int(input())
stones = list(map(int, input().split()))

nim_sum = 0

for stone in stones:
    if stone % 2 == 0:
        nim_sum ^= stone // 2 - 1
    else:
        nim_sum ^= stone // 2 + 1

if nim_sum != 0:
    print("koosaga")
else:
    print("cubelover")